scripts/zephyr_module.py

```diff
@@ -188,9 +188,9 @@ def main():
         # West is imported here, as it is optional (and thus maybe not installed)
         # if user is providing a specific modules list.
         from west.manifest import Manifest
-        from west.util import WestNotFound
+        from west.util import WestNotFound, west_topdir
         try:
-            manifest = Manifest.from_file()
+            manifest = Manifest.from_file(topdir=west_topdir())
```

Comment:
This is interesting.
I have just run a pdb on west, and when called as `Manifest.from_file()` it looks up `topdir` by itself, and that is working, so currently investigating what is actually happening.